tox.ini

```diff
@@ -42,6 +41,17 @@ commands =
     py.test -n {env:C7N_TEST_WORKERS:auto} --cov c7n --cov tools/c7n_azure/c7n_azure --cov tools/c7n_gcp/c7n_gcp --cov tools/c7n_kube/c7n_kube --cov tools/c7n_mailer/c7n_mailer tests tools {posargs}
 
 [testenv:py37]
+setenv =
+    AWS_DEFAULT_REGION=us-east-1
```

Comment:
originally I had just one environment value here with setenv but it overrides the rest, I wanted to experiment to see if passenv would work with the top level setenv to avoid the duplication, but overall it seems pretty minor for the gain, which is fairly large for the 2.7 and 3.6 runners.

Reply:
the additional issue with passenv is that it leaks the test execution in environment into the test which creates reliability issues. so afaics the duplication here gives us the least exposure to problems.